### PR TITLE
chore(endpoints): Update region tag with product prefix

### PR DESCRIPTION
**Pull request by arbrown**

### CODE REVIEW

**Comment by arbrown** [comment]
Freezing references to snippet in cl/534553109. Please do not merge until that is submitted (or let me take care of it.) Will un-freeze references once this PR has merged.

**Review by iennae — APPROVED** [review]
LGTM

**Review by rsamborski — APPROVED** [review]

**Pull request merged**
